### DISCUSSION TITLE
Django and Microsoft Visual Studio

### DISCUSSION THREAD

**Discussion by ArnaudNorel95** [community/community#23737] [New to GitHub]
<p>Hello, I use to develop with Microsoft Visual Studio and I want to test some projects found on Github, but I realize that if there is no .sln file, I can not run it on MVS.</p>
<p>Have you any idea how I can find projects on Github developed with MVS and so, with a .sln present in the files ? Does it exist a filter to find projects by languages and/or IDE used ?</p>
<p>(The reason why I don’t use VSCode is that I learned how to handle django with MVS and so I don’t know how to run my application on VSCode, each time I try to “python manage.py runserver” I got an error, even if I try with a project with totally works on MVS (where I just have to click button Run Web Server)</p>
<p>Thanks</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>You can search GitHub by topic to find results more suitable to you!</p>
<p><a href="https://help.github.com/en/github/searching-for-information-on-github/searching-topics" class="onebox" target="_blank">https://help.github.com/en/github/searching-for-information-on-github/searching-topics</a></p>
<p>Here’s a search for Visual Studio:</p>
<aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/search?q=Visual%2BStudio&amp;type=Topics" target="_blank">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/19803303/181079151-f39e073f-55c4-40d9-84c5-b4a9a7251baa.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://github.com/search?q=Visual%2BStudio&amp;type=Topics" target="_blank">Build software better, together</a></h3>

<p>GitHub is where people build software. More than 50 million people use GitHub to discover, fork, and contribute to over 100 million projects.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Alternatively, you could look into learning how to run any Django project in VS Code - this blogpost looks like a promising resource:</p>
<aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://dev-neha.blogspot.com/favicon.ico" class="site-icon" width="32" height="32">
      <a href="https://dev-neha.blogspot.com/2019/04/beginners-guide-starting-django-project.html" target="_blank">dev-neha.blogspot.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:276/145;"><img src="https://user-images.githubusercontent.com/19803303/181079152-77ace686-9d3c-4bfa-89d3-3d8700e11870.png" class="thumbnail" width="276" height="145"></div>

<h3><a href="https://dev-neha.blogspot.com/2019/04/beginners-guide-starting-django-project.html" target="_blank">Beginner's Guide:  Starting a Django Project in Visual Studio Code</a></h3>

<p>Django Project in Visual Studio Code</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
